### ISSUE TITLE
File Header Docblock Tag Order Nags?

### ISSUE THREAD

**Issue by robneu** [WordPress-Coding-Standards/WordPress-Coding-Standards#521]
It was recently [brought to my attention](https://github.com/wpsitecare/alpha/commit/82523072798b7499e9ba53347edbb59489404e49#commitcomment-15781317) that WPCS shouldn't be nagging about the `@subpackage` tag but on my end it is.

If the `@subpackage` tag is missing, I get a bunch of nags about the order of the various tags in the docblock and even if I re-arrange them to squelch most of those I wind up with a missing `@subpackage` tag.

I'm not sure if this an issue with Atom Linter's interpretation of the sniffs, or the sniffs themselves. Is anyone else having this problem?

**Comment by westonruter** [comment]
Can you share some example code that exhibits the issue?

I cannot reproduce the issue when running PHPCS 2.5.1 and `phpcs --standard=WordPress-Core,WordPress-Docs test.php` on:

```php
<?php
/**
 * Test
 *
 * @package Test
 */

/**
 * Foo do bar.
 */
function foo() {
	bar();
}
```

**Comment by GaryJones** [comment]
@robneu Any feedback on this?

**Comment by robneu** [comment]
I think this was some kind of a conflict with the Atom linter package using incorrect coding standards for the docblocks. I was never able to reproduce on another editor/CLI nor has the issue persisted after some updates to Atom's linter, so I think it's probably safe to just disregard.
